### PR TITLE
Ft/list players

### PR DESCRIPTION
**Pull request by wizlif**
Simulator Screen Shot - iPhone 13 - 2023-03-26 at 11 57 41

### CODE REVIEW

File: _packages/core/lib/models/player/player.dart [hunk 1]
```diff
@@ -14,33 +14,29 @@ class Player extends Equatable {
...
     required this.injured,
     required this.photo,
-    required this.number,
-    required this.position,
   });
 
   /// Creates a Player from Json map
   factory Player.fromJson(Map<String, dynamic> json) => Player(
-        id: json['id'] as String,
+        id: json['id'] as int,
```
Comment (jabahum) [inline _packages/core/lib/models/player/player.dart]: aint we goin to use  uid generated by firebase

File: _packages/core/lib/models/player/player.dart [hunk 2]
```diff
@@ -61,38 +57,30 @@ class Player extends Equatable {
...
-  final String weight;
+  final String? weight;
 
   /// A description for injured
   final bool injured;
 
   /// A description for photo
   final String photo;
 
-  /// A description for number
```
Comment (jabahum) [inline _packages/core/lib/models/player/player.dart:75]: i think we need this

**Review by jabahum — COMMENTED** [review]
looks good

File: _packages/core/lib/models/player/player.dart (hunk 1 above)
Comment (wizlif) [reply _packages/core/lib/models/player/player.dart]: Yes, it is a string

File: _apps/kapira_admin/lib/view/leagues/widgets/league_card.dart [hunk 3]
```diff
@@ -24,9 +26,17 @@ class LeagueCard extends StatelessWidget {
               color: context.kTheme.cardColor,
             ),
             padding: const EdgeInsets.all(10),
-            child: Image.asset(
-              league.logo,
-              package: 'assets',
+            child: CachedNetworkImage(
+              imageUrl: league.logo.image,
+              placeholder: (context, url) => ClipRRect(
+                child: ClipRRect(
```
Comment (jabahum) [inline _apps/kapira_admin/lib/view/leagues/widgets/league_card.dart:32]: clipRRect nested needs fix

File: _apps/kapira_admin/lib/view/teams/widgets/team_card.dart [hunk 4]
```diff
@@ -1,11 +1,13 @@
...
 import 'package:core/models/team/team.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_blurhash/flutter_blurhash.dart';
 
 class TeamCard extends StatelessWidget {
-  final Team league;
+  final Team team;
   final VoidCallback? onTap;
-  const TeamCard({super.key, required this.league, this.onTap});
+  const TeamCard({super.key, required this.team, this.onTap});
```
Comment (jabahum) [inline _apps/kapira_admin/lib/view/teams/widgets/team_card.dart:10]: need space

**Review by jabahum — APPROVED** [review]

**Comment by jabahum** [comment]
please look at these

**Pull request merged**